js/test/index.ts

```diff
@@ -30,6 +30,7 @@ import { createTransaction, parseURL } from '../src/wallet';
         '&memo=OrderId5678';
 
     const originalURL = NATIVE_URL;
+    console.log(originalURL);
```

Comment:
We can move this file since it's not a real test.

Reply:
I was thinking of converting it into a test. thoughts?

or keep it as is and move it to `/example` directory?

Reply:
I think we can keep it as is for now, extract the behavior to tests, and decide what to do with it later.